mfr/core.py

```diff
@@ -370,3 +371,26 @@ def collect_static(dest=None, dry_run=False):
                 .format(**locals()))
         else:
             copy_dir(static_path, namespaced_destination)
+            template_path = os.path.join(namespaced_destination, 'templates/')
+            # If there are templated static files that once had hard-coded dependencies
+            if os.path.isdir(template_path):
+                for dirpath, dirnames, filenames in os.walk(template_path):
+                    # Walk through each template
+                    for template in filenames:
+                        with open(os.path.join(template_path, template)) as src:
+                            # Resolve the dependency
+                            data = re.sub('{{STATIC_PATH}}',
+                                '/static'+namespaced_destination.split('static')[-1],
```

Comment:
This will not work if the user configures mfr.config['STATIC_PATH'] to anything other than `/static`. Why not replace `{{STATIC_PATH}}` with the config variable?